packages/synapse-react-client/src/lib/containers/row_renderers/Dataset.tsx

```diff
@@ -31,7 +34,14 @@ class Dataset extends React.Component<DatasetProps, never> {
     }
 
   public render() {
-    const { data, schema, selectColumns, columnModels } = this.props
+    const {
+      data,
+      schema,
+      selectColumns,
+      columnModels,
+      genericCardSchema,
+      queryVisualizationContext: { getColumnDisplayName },
```

Comment:
Need the queryVisualizationContext to have access to the getColumnDisplayName (which possibly transforms the camel case column name into a more friendly version)